qualtran/bloqs/arithmetic/addition.py

```diff
@@ -96,7 +99,7 @@ def add_my_tensors(
     ):
         import quimb.tensor as qtn
 
-        N = 2**self.bitsize
+        N = 2**self.dtype.bitsize
```

Comment:
I think this tensor only works for unsigned ints, no?

xref #691

Reply:
I raise an error for now.